CoreFoundation/RunLoop.subproj/CFMessagePort.c

```diff
@@ -496,8 +496,8 @@ static CFMessagePortRef __CFMessagePortCreateRemote(CFAllocatorRef allocator, CF
     ctx.retain = NULL;
     ctx.release = NULL;
     ctx.copyDescription = NULL;
-    task_get_bootstrap_port(mach_task_self(), &bp);
-    native = (KERN_SUCCESS == ret) ? CFMachPortCreateWithPort(allocator, port, __CFMessagePortDummyCallback, &ctx, NULL) : NULL;
+    ret = task_get_bootstrap_port(mach_task_self(), &bp);
```

Comment:
These couple of lines may need a quick eyeball by @phausler -- the direct context in the GitHub view doesn't necessarily determine that these changes are valid. What was the warning that you were trying to fix here? Unused return or local variable?

Reply:
So looking at the changes they look sensible in the context of this change, but it would be good to see if the fixes can be upstreamed efficiently.

Reply:
I think they were both reads before being initialised, but the `port` -> `bp` fix is more of an educated guess  :)

Reply:
I don't think this is what you want actually. This function really shouldn't exist at all in this environment.

The core idea here is to look up the port by name but that part is missing. We should probably just replace the entire thing with `return NULL`.

Reply:
If we want to keep the rest of the code for some reason (documentation purposes?) then we should not bother with `task_get_bootstrap_port` and just make sure `port == NULL` for the rest of the function.

Reply:
If the function can be replaced with a `return NULL` then it would be better to just do that. The old body will still be in git for the curious.